packages/cf-typography/src/cf-typography.less

```diff
@@ -24,35 +24,35 @@
 // Running text elements
 
 // .a-micro-copy
-@micro-copy:                    @black;
+@micro-copy: @black;
```

Comment:
Matches indenting of variables at the top of the file https://github.com/cfpb/capital-framework/blob/master/packages/cf-typography/src/cf-typography.less#L11